main.js

```diff
@@ -0,0 +1,63 @@
+// create animal template
+
+class Animal {
+   constructor(name, happiness) {
+      this.name = name;
+      this.happiness = happiness;
+   }
+   feed() {
+      //print to DOM
+      console.log(`${this.name} loves feeding time!`);
+      // increase happiness level
+      return this.happiness += 20;
+   }
+   play() {
+      //print to DOM
+      console.log(`${this.name} loves playing!`);
+      // increase happiness level
+      return this.happiness += 15;
+   }
+   wash() {
+      //print to DOM
+      console.log(`Uh-oh! ${this.name} hates bath time.`);
+      // increase happiness level
+      return this.happiness -= 15;
+   }
+   stroke() {
+      //print to DOM
+      console.log(`${this.name} loves cuddles!`);
+      // increase happiness level
+      return this.happiness += 5;
+   }
+}
+
+class Dog extends Animal {
+   constructor(name, happiness) {
+      super(name, happiness)
+   }
+   get happinessUnit() {
+      return `${this.happiness} tail wags per minute`;
```

Comment:
Do you think you could possibly incorporate some more specific properties that each subclass could have? So that each animal has a distinct behaviour?

Reply:
I know you have your individual return statements but I'd like to see some properties that are unique to each type of animal.